wallet_anchor.go

```diff
@@ -77,11 +77,17 @@ func (l *LndRpcWalletAnchor) FundPsbt(ctx context.Context, packet *psbt.Packet,
 		}
 	}
 
+	// We'll convert the fee rate to sat/vbyte as that's what the FundPsbt
+	// expects. We add 999 to round up to the nearest 1000 to prevent issues
+	// where the fee doesn't meet the min_relay_fee because of rounding
+	// down.
+	satPerVByte := uint64((feeRate.FeePerKVByte() + 999) / 1000)
```

Comment:
How about just: `uint64(math.Ceil(float64(feeRate.FeePerKVByte()) / 1000))`?

Reply:
You like that more? I thought this was pretty elegant. It would have prevented an extra import if it weren't for the fact that we already use `math`. It also doesn't need the cast to float, which is obviously a huge performance gain. :wink:

Reply:
Sure, it is elegant. But I tend to always go with the more clear and easy to understand version. IMO with the "elegant" version you need the comment above (why are we adding 999?) while `math.Ceil()` by definition/name says: Rounding up.
But not going to block the PR on it...

Reply:
I'm with Oli on this. I need to think less with `math.Ceil`. But Gij's ` + 999` is way more elegant. But I love to think less.